### ISSUE TITLE
PCB does not fit 200A shunt

### ISSUE THREAD

**Issue by magpern** [stuartpittaway/diyBMS-CurrentShunt#2]
The 200A shunt does not fit on the pcb.
The holes on the shunt is of by about half a hole.
And the shunt covers the pcb corner holes.

This is the shunt, I think it is the correct one.
https://www.digikey.se/short/bd507d49

![20210703_234002](https://user-images.githubusercontent.com/6589760/124367999-e7dc2b80-dc5c-11eb-9a40-c44d9dadc97b.jpg)
![20210703_233959](https://user-images.githubusercontent.com/6589760/124368006-ed397600-dc5c-11eb-9e57-37fd769a3818.jpg)
![20210703_233938](https://user-images.githubusercontent.com/6589760/124368036-15c17000-dc5d-11eb-91cf-50a0249b346a.jpg)

**Comment by stuartpittaway** [comment]
Ok, I'll update the PCB. There are no components/traces on that side of the board, so you should be safe to drill a new hole.

**Comment by stuartpittaway** [comment]
Can I ask you to confirm these measurements against the real shunt you have?

It should be 82.5mm by 44.4mm


![image](https://user-images.githubusercontent.com/1201909/124576679-7232b400-de44-11eb-9e41-0b5890cc3ed6.png)

**Comment by magpern** [comment]
I finally found my slide measure tool. My son tidied the garage.. garage looks nice.. tools are a little random.

The outer dimensions of the actual shunt (200A) is pretty must correct with the picture (+/- 1 mm)
But the hole placement is not.
I used a digital slide measure and got the holes to 5.3mm, which is also proven by the other measurements, hole-side to hole-side, and outer-side to hole. I measured several times and got almost the same numbers.

hole is measured physically and I got 5.3mm which corresponds to C-B divided by 2

A = 3.7 mm
B = 26.6 mm
C = 37.2 mm
 D = 39.0 mm

![image](https://user-images.githubusercontent.com/6589760/124666119-d9388300-dead-11eb-9ffe-64e01bf352f1.png)
